pkg/scannerv4/mappers/mappers.go

```diff
@@ -695,6 +770,47 @@ func pkgFixedBy(enrichments map[string][]json.RawMessage) (map[string]string, er
 	return pkgFixedBys, nil
 }
 
+// cveEPSS unmarshals and returns the EPSS enrichment, if it exists.
+func cveEPSS(enrichments map[string][]json.RawMessage, ctx context.Context) (map[string]epssDetail, error) {
+	enrichmentList := enrichments[epss.Type]
+	if len(enrichmentList) == 0 {
+		zlog.Warn(ctx).
+			Str("enrichments:", epss.Type).
+			Msg("no EPSS enrichments found")
+		return nil, nil
```

Comment:
You have a line below saying `return nil, nil` which indicates the same condition as the Warn() you're logging here.

Reply:
Maybe consider putting this outside the function.

Also, just to confirm, should this be a warning?